packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -413,7 +413,30 @@ const schema: SchemaType<DbPost> = {
     editableBy: ['admins', 'sunshineRegiment'],
     group: formGroups.adminOptions,
   },
-  
+
+  // By default, the read time for a post is calculated automatically from the word count.
+  // Sometimes this incorrect (often due to link posts, videos, etc.) so it can be overridden
+  // manually by setting this field.
+  readTimeMinutesOverride: {
```

Comment:
I was able to submit a negative number, but since it's admin only it's probably fine